applications/nrf_desktop/doc/dev_descr.rst

```diff
@@ -0,0 +1,28 @@
+.. _dev_descr:
+
+Device Description
```

Comment:
```suggestion
Device description module
```
Now, most of the module page titles have `module` included, so let's stick to this style.

Reply:
ok, changed